locales.php

```diff
@@ -1787,19 +1902,19 @@ public function GP_Locales() {
 		$zh->native_name = '中文';
 		$zh->lang_code_iso_639_1 = 'zh';
 		$zh->lang_code_iso_639_2 = 'zho';
-		$zh->country_code = '';
 		$zh->slug = 'zh';
 		$zh->nplurals = 1;
 		$zh->plural_expression = '0';
 
 		foreach( get_defined_vars() as $locale ) {
-			$this->locales[$locale->slug] = $locale;
+			$this->locales[ $locale->slug ] = $locale;
 		}
 	}
 
-	public static function instance() {
-		if ( !isset( $GLOBALS['gp_locales'] ) )
+	public static function &instance() {
```

Comment:
Correct me if I'm wrong but If I read [this here](http://php.net/references.return) correctly, when you introduce the ampersand here, you need to change all assignments into reference assignments, too. I would actually refrain from adding the ampersand.